server/src/modules/models/video/handler.js

```diff
@@ -1,30 +1,41 @@
 const eventEmitter = require('../../../event-manager').getInstance();
 const { VIDEO_QUEUE_EVENTS } = require('../../queues/constants');
-const { updateHistory } = require('./service');
+const { updateHistory, update } = require('./service');
+const { VIDEO_STATUS } = require('../../db/constant')
 
 const setup = () => {
   // eventEmitter.on(VIDEO_QUEUE_EVENTS.VIDEO_UPLOADED, (data) => {
   //   console.log('VIDEO_QUEUE_EVENTS.VIDEO_UPLOADED Event handler', data);
   // });
-  console.log('registering video queue events');
+
   const SERVER_URL = process.env.SERVER_URL;
+
   Object.values(VIDEO_QUEUE_EVENTS).forEach((eventName) => {
+
     eventEmitter.on(eventName, async (data) => {
-      console.log(`models/video/handler.js - ${eventName}`, data);
+
       if (eventName === VIDEO_QUEUE_EVENTS.VIDEO_PROCESSED) {
         await updateHistory(data.id, {
           history: { status: eventName, createdAt: new Date() },
           processedPath: data.path,
         });
         return;
       }
+
       if (eventName === VIDEO_QUEUE_EVENTS.VIDEO_HLS_CONVERTED) {
         await updateHistory(data.id, {
           history: { status: eventName, createdAt: new Date() },
           hlsPath: data.path,
         });
+        
+        await update({ 
+          _id: data.id,
+          status: VIDEO_STATUS.PUBLISHED
+        });
```

Comment:
We can merge these two operations into a single update command. But for now, it is ok. 👍